scripts/mainnet/deploy_child.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python3
+
+from ape import project
+
+from deployment.constants import (
+    CONSTRUCTOR_PARAMS_DIR,
+    OZ_DEPENDENCY,
+)
```

Comment:
not a relevant thing, but OZ_DEPENDENCY is not actually used.

```suggestion
from deployment.constants import CONSTRUCTOR_PARAMS_DIR
```